internal/goldendataset/metric_gen.go

```diff
@@ -54,7 +54,7 @@ func DefaultCfg() MetricCfg {
 		NumILMPerResource:    1,
 		NumMetricsPerILM:     1,
 		NumPtLabels:          1,
-		NumPts:               1,
+		NumPtsPerMetric:      1,
```

Comment:
In the future, these kind of changes can be split in small trivial to review PRs :)